src/backend/core/tests/teams/test_core_api_teams_create.py

```diff
@@ -2,6 +2,11 @@
 Tests for Teams API endpoint in People's core app: create
 """
 import pytest
+from rest_framework.status import (
```

Comment:
Imo it's better to use raw values in tests:
- tests should be as simple and understandable as possible
- If the value of HTTP_401_AUORIZED changes one day, I would like my tests to break and decide what to do... 

ok, the change in value may never happen here... but I assume that what drives this change is that we would want to replace all other raw values by constants in our tests and I think it's an anti-pattern.

Reply:
except in tests yes. Not going to bikeshed about it though.